com.bladecoder.adventure-editor.json

```diff
@@ -0,0 +1,79 @@
+{
+    "app-id" : "com.bladecoder.adventure-editor",
+    "runtime" : "org.freedesktop.Platform",
+    "runtime-version" : "1.6",
+    "sdk" : "org.freedesktop.Sdk",
+    "sdk-extensions" : [
+        "org.freedesktop.Sdk.Extension.openjdk9"
+    ],
+    "command" : "adventure-editor",
+    "finish-args" : [
+        "--share=network",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
Does this actually work? I don't know if Java has wayland support out of the box.

Reply:
It works in wayland, I don't know if it's using the X11 compatibility layer

Reply:
> It works in wayland, I don't know if it's using the X11 compatibility layer

 It surely doesn't. In GNOME-Shell you can hit alt+f2, run `lg`, then in windows click on one to see if it is an X11Window.

Reply:
![imagen](https://user-images.githubusercontent.com/50350/36300764-bcdfe27c-1302-11e8-8f14-b8c51f786f25.png)

All my windows have the GType:MetaWindowX11.

Reply:
So it doesn't support Wayland. Can you remove it? We will give it last test build and merge.

Reply:
Done